src/test/java/it/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMIT.java

```diff
@@ -134,6 +167,10 @@ public void testPostBuildStatus() throws Exception {
     }
 
     private BitbucketSCM createSCM(String... refs) {
+        return createCustomRepoSCM(REPO_NAME, REPO_SLUG, refs);
+    }
+
+    private BitbucketSCM createCustomRepoSCM(String repoName, String repoSlug, String... refs) {
```

Comment:
Is this okay? Having the two explicit args followed by a vararg of the same type feels a bit confusing

Reply:
I think its fine